http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -76,7 +76,17 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
       )
 
       // Get resources from landing zone
-      landingZoneResources = getLandingZoneResources
+      workspaceId <- F.fromOption(
+        dbApp.cluster.workspaceId,
+        AppCreationException(
+          s"Workspace ID not found in DB for app ${app.appName.value}",
+          Some(ctx.traceId)
+        )
+      )
+      petToken = Authorization(
+        Token(AuthScheme.Bearer, "sam-UAMI-token") // TODO get real token pet token. Add required SAM API
```

Comment:
Yup we need to figure out how to get a UAMI token -- so let's just leave this TODO for now.

Reply:
Using the UAMI token came up [here](https://github.com/DataBiosphere/leonardo/pull/2925#discussion_r1012218356) as well. I should be able to carve out some time tomorrow to look at this